dynamodb-enhanced/runtime/src/main/java/io/quarkiverse/amazon/dynamodb/enhanced/runtime/DynamodbEnhancedClientRecorder.java

```diff
@@ -59,20 +59,23 @@ private DynamoDbEnhancedClientExtension createExtension(String extensionClassNam
         Class<?> clazz = null;
         try {
             clazz = Class.forName(extensionClassName, false, Thread.currentThread().getContextClassLoader());
-            // try builder pattern in aws sdk
-            Method builderMethod = clazz.getMethod("builder");
-            Object builder = builderMethod.invoke(null);
-            DynamoDbEnhancedClientExtension extension = (DynamoDbEnhancedClientExtension) builder.getClass().getMethod("build")
-                    .invoke(builder);
-            return extension;
+            try {
+                Method builderMethod = clazz.getMethod("builder");
+                Object builder = builderMethod.invoke(null);
+                return (DynamoDbEnhancedClientExtension) builder.getClass().getMethod("build").invoke(builder);
+            } catch (NoSuchMethodException e) {
+                Method createMethod = clazz.getMethod("create");
+                return (DynamoDbEnhancedClientExtension) createMethod.invoke(null);
+            }
         } catch (ClassNotFoundException | IllegalAccessException | InvocationTargetException e) {
             LOG.error("Unable to create extension " + extensionClassName, e);
             return null;
         } catch (NoSuchMethodException e) {
             try {
                 return (DynamoDbEnhancedClientExtension) clazz.getDeclaredConstructor().newInstance();
-            } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException ex) {
-                LOG.error("Unable to create extension " + extensionClassName, e);
+            } catch (InstantiationException | IllegalAccessException | InvocationTargetException |
+                     NoSuchMethodException ex) {
+                LOG.error("Unable to create extension " + extensionClassName, ex);
```

Comment:
```suggestion
            } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException ex) {
                LOG.error("Unable to create extension " + extensionClassName, ex);
```

Be sure to run `install` goal or `mvn -f runtime net.revelc.code.formatter:formatter-maven-plugin:2.26.0:format`